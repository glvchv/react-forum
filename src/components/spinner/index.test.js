import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '.';


describe('Spinner component', () => {
    it('should render component', () => {
        const tree = renderer.create(
            <Spinner />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});