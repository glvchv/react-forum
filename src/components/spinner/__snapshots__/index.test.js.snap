// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should render component 1`] = `
<div
  className="container"
>
  <img
    alt="loading..."
    className="gif"
    src="/spinner.gif"
  />
</div>
`;
